x-pack/plugins/siem/public/resolver/store/camera/reducer.ts

```diff
@@ -19,7 +19,7 @@ import { scaleToZoom } from './scale_to_zoom';
  */
 export function cameraInitialState(): CameraState {
   const state: CameraState = {
-    scalingFactor: scaleToZoom(1), // Defaulted to 1 to 1 scale
+    scalingFactor: scaleToZoom(1.5), // Defaulted to 1 to 1 scale
```

Comment:
So nodes start off scaled in enough to show submenus

Reply:
I'm a little skeptical of this change. This means that when working on the UI, we'll have to remember that 1 == 1.5. In other words, if a mock has a 10 pixel box, we'll have to remember that we need to make it 6.667px instead.

I'm not necessarily opposed to this, but I'd like to hear your thoughts on it. 
Also, if you keep this change, could you please update the comment.

Reply:
I will probably withdraw this PR because it won't land in time to help the user test. Regarding your comment here, I think all the mocks are made within the boundaries of the Resolver canvas are understood to be inherently relative to other components in the canvas (because it's scalable in every configuration we've shown it in), so "10 pixels" wouldn't really be a stable unit of measurement. If we think that might cause confusion or weird constants to pop up, we might be well served by adding a comment to that effect here. ("Defaulted to 1 to 1 scale" doesn't really explain that changing it would have those kinds of effects.)